operator/config/manifests/bases/rhacs-operator.clusterserviceversion.yaml

```diff
@@ -563,12 +563,10 @@ spec:
       - description: Custom environment variables to set on managed pods' containers.
         displayName: Environment Variables
         path: customize.envVars
-      - description: To ensure comprehensive monitoring of your cluster activity,
-          Red Hat Advanced Cluster Security will run services on every node in the
-          cluster, including tainted nodes by default. If you do not want this behavior,
-          please select 'AvoidTaints' here.
-        displayName: Taint Toleration
-        path: perNode.taintToleration
+      - description: Settings for the Node-Scanner container, which is responsible
```

Comment:
I'm afraid the container is called `node-inventory` and we'll have to follow that unless you're open to perform a big rename.

Reply:
That is correct, unfortunately.
While the container itself is name `node-inventory` in code, we named the user-facing part of this feature `Node Scanning`, in log messages and Helm Chart values.
As our docs also call it Node Scanner ([here in 2.b](https://docs.openshift.com/acs/4.0/operating/manage-vulnerabilities/scan-rhcos-node-host.html#rhcos-enable-node-scan_scan-rhcos-node-host)), I decided to go with the same verbiage and call it Node Scanner container.

It's a tough one and I think we should discuss in the feature team how we can improve this split of node-inventory/node-scanner, as it might be confusing for users.
What do you think? Should I rename the help text to the container name (`node-inventory`), or keep it `Node Scanner` to adhere to docs and logs verbiage?

Reply:
The problem with that docs page is that it mentions both `Node-inventory` in the text (I counted 5 times) and `NodeScanner` (once). This should be definitely discussed and streamlined, I think.

Reply:
I think when we refer to the container, we should call it `node-inventory` as this is its name. Node Scanning was meant to be the name of the feature and should be used in all other cases.

As an alternative, we may phrase it in a way to show that we speak about the same: _... Node Scanner (node-inventory container)..._

Reply:
Resolving comment by adding [0296560](https://github.com/stackrox/stackrox/pull/6160/commits/0296560e291e5a758622ace57dcc4d7ec9ab00ba)